### PR TITLE
[don't merge yet]

### PR DESCRIPTION
**Pull request by sebastian-lerner**
update for new preview resource classes

### CODE REVIEW

File: jekyll/_includes/snippets/gpu-linux-resource-table.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Class | vCPUs | RAM | GPUs | GPU model | GPU Memory (GiB) | Disk Size (GiB) | Cloud | Server
 ---|---|---|---|---|---|---|---|---
+gpu.nvidia.small |   4   | 16  | 1 | Nvidia Tesla P4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.gen2 |   4   | 16  | 1 | Nvidia A10G | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.multi |   4   | 15  | 2 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.medium.multi |   8   | 30  | 4 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
 gpu.nvidia.medium |   8   | 30  | 1 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
```
Comment (brentmmarks) [inline jekyll/_includes/snippets/gpu-linux-resource-table.md:7]: think the disc size for all gpus right now is 150 -> do we need to update these?

Comment (sebastian-lerner) [reply jekyll/_includes/snippets/gpu-linux-resource-table.md:7]: can you confirm it's 150? if yes, i'll update

Comment (brentmmarks) [reply jekyll/_includes/snippets/gpu-linux-resource-table.md:7]: Default is 150: https://github.com/circleci/machine-provisioner/blob/c365d9101509a29fa7dc27e7dbcd7683313da31a/provider/provider.go#L12

We dont specify an override here like we do for windows disk stuff: https://github.com/circleci/machine-provisioner/blob/main/deploy/conf/machine-provisioner-config.yml#L920

File: jekyll/_includes/snippets/gpu-linux-resource-table.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 Class | vCPUs | RAM | GPUs | GPU model | GPU Memory (GiB) | Disk Size (GiB) | Cloud | Server
 ---|---|---|---|---|---|---|---|---
+gpu.nvidia.small |   4   | 16  | 1 | Nvidia Tesla P4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.gen2 |   4   | 16  | 1 | Nvidia A10G | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
```
Comment (brentmmarks) [inline jekyll/_includes/snippets/gpu-linux-resource-table.md:4]: also gpu memory is 24 based on g5 docs: https://aws.amazon.com/ec2/instance-types/g5/

**Review by brentmmarks — APPROVED** [review]

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**